10.0 pos partner sync (#939)
it-projects-llc/pos-addons: pos_partner_sync/README.rst

   :alt: License: LGPL-3

======================
 Sync Partners in POS
======================

> The module allows to instantly update partner data in opened POSes via longpolling connection if a predefined partner field has been changed in the backend.

Reviewer: ```suggestion
The module allows to instantly update partner data in opened POSes via longpolling connection if any of predefined partner fields have been changed in the backend.
```

> By default, Odoo updates partner data when open the client list. This module doesn't require an extra action for that and updates partner data in POSes instantly after the predefined partner field has been modified.

Reviewer 2: I entered the client list before I scanned the new customer barcode from other POS. Otherwise, POS can not find the barcode.

Author: Had you specified the barcode field to be synchronized? You can find how to do it in the documentation.

Author: Confirm. That's the issue

Reviewer 2: Ok, resolved

   :alt: License: LGPL-3

======================
 Sync Partners in POS
======================

> The module allows to instantly update partner data in opened POSes via longpolling connection if partner barcode has been changed in the backend.

Reviewer: Not only barcode
